src/components/UserDetails/UserDetails.js

```diff
@@ -1,44 +1,140 @@
 import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import './userDetails.scss';
+import { Row, Col } from 'reactstrap';
 import PropTypes from 'prop-types';
+import './userDetails.scss';
+import Input from '../Inputs/Input';
+import Button from '../Button';
 
-const UserDetails = ({ profile }) => {
+const UserDetails = ({ profile, toggleEdit, isEdit, handleInput, submitProfile }) => {
   const getValue = (value) => {
```

Comment:
This is looks more succinct 👇 
```js
const getValue = (value) => (profile && profile[value]) ? profile[value] : 'Nil';
```

Also this might work 👇 

```js
const getValue = (value) =>  profile[value] || 'Nil';
```